test/node_modules/duplicate_symbols/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "duplicate_symbols",
+  "version": "0.0.0",
+  "description": "Duplicate Symbols Test",
+  "main": "index.js",
+  "private": true,
+  "dependencies": {
+    "bindings": "~1.2.1"
+  },
+  "scripts": {
+    "test": "node index.js"
+  },
+  "gypfile": true
```

Comment:
I realize you copied this from the other package.json but it's basically a no-op. npm auto-detects *.gyp files.